assets/tests/components/nav/topNav.test.tsx

```diff
@@ -68,47 +68,55 @@ describe("TopNav", () => {
       )
 
       expect(
-        result.queryByRole("button", { name: "User Info" })
+        screen.queryByRole("button", { name: "User Info" })
       ).toBeInTheDocument()
     })
 
-    test("brings up an element with 'logged in as' text when clicked", async () => {
+    test("brings up an element with 'logged in as' text and a logout ling when clicked", async () => {
       jest.mocked(getTestGroups).mockReturnValue([TestGroups.KeycloakSso])
 
       const dispatch = jest.fn()
       const user = userEvent.setup()
-      const result = render(
+      render(
         <StateDispatchProvider state={initialState} dispatch={dispatch}>
           <BrowserRouter>
             <TopNav />
           </BrowserRouter>
         </StateDispatchProvider>
       )
 
-      expect(result.queryByText("Logged in as")).not.toBeInTheDocument()
+      expect(screen.queryByText("Logged in as")).not.toBeInTheDocument()
 
-      await user.click(result.getByTitle("User Info"))
+      await user.click(screen.getByTitle("User Info"))
 
-      expect(result.queryByText("Logged in as")).toBeInTheDocument()
+      expect(screen.queryByText("Logged in as")).toBeInTheDocument()
+
+      const logoutLink = screen.queryByRole("link", { name: "Log out" })
+
+      expect(logoutLink).toBeInTheDocument()
```

Comment:
nit(non-blocking): I'd prefer this to be `toBeVisible()`, it's essentially like `toBeInTheDocument`, but if we ever get CSS into our jest tests, we'll get better tests for "free".
```suggestion
      expect(logoutLink).toBeVisible()
```